app/views/partials/_sidemenu.html.erb

```diff
@@ -1,16 +1,26 @@
   <div class="sidenav">
       <div class="sidenav-logo sidenav-item">
-        <i class="fab fa-algolia sidenav-icon"></i>Logo
+        <%= image_tag('AlotLogo/logo.png', alt: "Edit Entry") %>
       </div>
       <div class="sidenav-home sidenav-item">
-        <i class="fas fa-home sidenav-icon"></i>Home
+        <%= image_tag('icons/Home Icon.png', alt: "Edit Entry") %> Home
       </div>
       <div class="sidenav-list sidenav-item">
-        <a href="#about"><i class="fab fa-alipay sidenav-icon"></i>Device Manage</a>
-        <a href="#services"><i class="fab fa-alipay sidenav-icon"></i>Air Handing Unit</a>
-        <a href="#clients"><i class="fab fa-alipay sidenav-icon"></i>Data Dashboard</a>
-        <a href="#contact"><i class="fab fa-alipay sidenav-icon"></i>API Modules</a>
+        <a href="#about">
+          <%= image_tag('icons/Item Icon.png', alt: "Edit Entry") %> Device Manage
```

Comment:
remove the `alt` from all image_tags and set the height to 20px so it align nicely with the text
```erb
<%= image_tag('icons/Item Icon.png', height: '20') %>